plugins/scaffolder-backend/package.json

```diff
@@ -81,12 +81,13 @@
     "isomorphic-git": "^1.23.0",
     "jsonschema": "^1.2.6",
     "knex": "^2.0.0",
+    "libsodium-wrappers": "^0.7.11",
```

Comment:
Hm this adds almost 200 kB. Is there any way we can use the builtin Crypto module in node or so?

Reply:
Potentially?  Just following GitHub doco directly:
- https://docs.github.com/en/rest/actions/secrets?apiVersion=2022-11-28#create-or-update-a-repository-secret
- https://docs.github.com/en/actions/security-guides/encrypted-secrets

Thoughts?

Reply:
Ah I think that makes a decent argument for keeping it as is.